websites/B/BeastFM/dist/metadata.json

```diff
@@ -9,14 +9,13 @@
     "en": "BeastFM is a US radio station with the #1 hits 24/7"
   },
   "service": "BeastFM",
-  "version": "1.0.3",
-  "logo": "https://i.imgur.com/l6meesB.png",
-  "thumbnail": "https://i.imgur.com/tWSpYUK.png",
-  "color": "#ff8300",
+  "version": "1.0.4",
+  "logo": "https://i.imgur.com/UtOgutm.png",
+  "thumbnail": "https://i.imgur.com/hLHoPD4.png",
+  "color": "#685884",
```

Comment:
the thumbnail is empty? Use a promotional banner or a picture of the website homepage.

Reply:
No, it is the gradient because we don't have a banner.

Reply:
Then take a screenshot of the website homepage...

Reply:
Yeah it's not updated yet we are switching host and the other one would not update and now the website is down.

Reply:
Alright make a new pr then when you can fix it.